test/release/examples/benchmarks/shootout/fasta.chpl

```diff
@@ -96,47 +101,93 @@ proc repeatMake(desc, alu, n) {
 }
 
 //
-// Output a random sequence of length 'n' using distribution a
+// Output a random sequence of length 'n' using distribution 'a'
```

Comment:
Just to sanity-check this comment: What does "distribution 'a'" refer to? I don't see a variable named 'a' anywhere.